python/ray/tune/trainable.py

```diff
@@ -659,3 +661,16 @@ def _export_model(self, export_formats, export_dir):
             A dict that maps ExportFormats to successfully exported models.
         """
         return {}
+
+    def _import_model(self, import_format, import_file):
```

Comment:
@richardliaw are you OK with these changes to Tune? Wasn't clear from the previous comment chain.

Reply:
Ah sorry about this slow response. `import_model` shouldn't be in Tune I think.

Reply:
I took it out.